src/Uno.UI/UI/Xaml/Media/Animation/Animators/GPUFloatValueAnimator.iOSmacOS.cs

```diff
@@ -37,10 +38,10 @@ internal class GPUFloatValueAnimator : IValueAnimator
 		private float _to;
 		private float _from;
 		private long _duration;
-		private IEnumerable<IBindingItem> _bindingPath;
-		private FloatValueAnimator _valueAnimator;
-		private UnoCoreAnimation _coreAnimation;
-		private IEasingFunction _easingFunction;
+		private IEnumerable<IBindingItem>? _bindingPath;
```

Comment:
Can this really be null? It looks like it's set in the constructor?

Reply:
We are making it null on the dispose method https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/Media/Animation/Animators/GPUFloatValueAnimator.iOSmacOS.cs#L557. Should we change it there?

Reply:
Ah, missed that line. In that case, I think Dispose should do `_bindingPath = null!` and keep the field non-nullable. Attempts to operate on a disposed object look fundamentally wrong and may be the actual root cause.

**However,** depending on how complex is it to fix the use-after-dispose issue, it may still be acceptable to tolerate nulls. It may be worth adding a comment though

Reply:
Updated. Making the field non-nullable and validating for null on the constructor should work